src/content/getting-started/architecture/gateway-engine/_index.en.md

```diff
@@ -55,7 +55,42 @@ which traffic can be routed.
 ![Figure 1 - High Availability in stable cluster](/images/high-availability/HA_Cluster1.png)
 <!-- Image Source: https://docs.google.com/presentation/d/180CtHZnr9PP5Rh98VEmkQz3ovc5AGXG9wosoHMLhgaY/edit -->
 
-#### Gateway Failover
+### Cable Drivers Topology Overview
+
+#### Libreswan
+
+The following diagram shows a high level topology for a Submariner deployment created with:
+
+```bash
+make deploy using=lighthouse
+```
+
+In this case, Libreswan is configured to create 4 IPsec tunnels to allow for:
+
+* Pod subnet to Pod subnet connectivity
+* Pod subnet to Service subnet connectivity
+* Service subnet to Pod subnet connectivity
+* Service subnet to Service subnet connectivity
+
+![Figure 2 - IPsec Tunnel Mode inter connected clusters](/images/cable-drivers/ipsec_cable.png)
+
+#### VXLAN
+
+The following diagram shows a high level topology for a Submariner deployment created with:
+
+```bash
+make deploy using=lighthouse, vxlan
+```
+
+With the VXLAN cable driver routes in table 100 are used on the source Gateway to steer the traffic
+into the vxlan-tunnel interface.
+
+The figure shows a simple interaction (a ping from one pod in one cluster to another pod in
```

Comment:
Currently the diagram does not show how the ping traverses, do you want to include some numbers in the diagram to show how the packet traverses?

Reply:
Sure I can do that